gradle.properties

```diff
@@ -1,5 +1,5 @@
 # Project-wide Gradle settings.
 
-#GCM_API_KEY="change here"
-DIMELO_SDK_SECRET="change here"
-DIMELO_SDK_DOMAIN_NAME="change here"
+ENGAGE_DIGITAL_MESSAGING_SECRET="93e22debd62844e98d2dd2b7c6f2916160e7903ee806ff51d67a7b5b6ef31878"
```

Comment:
```suggestion
ENGAGE_DIGITAL_MESSAGING_SECRET="change_here"
```

Reply:
this is a public repository we should never commit secret in it

Reply:
done